en_US.md

```diff
@@ -12,16 +12,16 @@ Serving a company that encourages the "996" working pattern usually means workin
 
 ## Gaining more popularity
 
-In early 2019, a Chinese E-commerce company called _Youzan_ has announced the adoption of the "996" working pattern in the future, at the company's Chinese New Year Party. The CEO of Youzan responds: "This would definitely be a good decision when we look back in a few years time.".
+In early 2019, a Chinese E-commerce company called _Youzan_ has announced the adoption of the "996" work schedule in the future, at the company's Chinese New Year Party. The CEO of Youzan responds: "This would definitely be a good decision when we look back in a few years time.".
```

Comment:
```suggestion
In early 2019, a Chinese E-commerce company called _Youzan_ announced the adoption of the "996" work schedule in the future, at the company's Chinese New Year Party. The CEO of Youzan responded: "This would definitely be a good decision when we look back in a few years time.".
```